### PR TITLE
Publish the marker inside the fast folder

### PR DESCRIPTION
**Pull request by upodroid**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
latest-fast.txt marker should be stored inside the fast folder

#### Which issue(s) this PR fixes:
xref: https://github.com/kubernetes-sigs/release-sdk/pull/291
Fixes: https://github.com/kubernetes/release/issues/3403

#### Does this PR introduce a user-facing change?

```release-note
breaking change: fast marker (latest-fast.txt) is now stored inside the fast folder instead of the gcsRoot folder(ci)
```

### CODE REVIEW

**Comment by upodroid** [comment]
this is working as expected
```
ubuntu@dev:~/go/src/k8s.io/kubernetes$ krel ci-build --bucket kubernetes-staging-a1db298f53-us-central1 --allow-dup=false --fast
INFO Build type has been set to ci
INFO Bucket has been set to kubernetes-staging-a1db298f53-us-central1
INFO GCS root has been set to ci
INFO Current working directory: /home/ubuntu/go/src/k8s.io/kubernetes
INFO Getting workspace status
STABLE_BUILD_GIT_COMMIT 1850794626bb995bb754d54be3328c86ee880ba5
STABLE_BUILD_SCM_STATUS clean
STABLE_BUILD_SCM_REVISION v1.29.0-alpha.2.1554+1850794626bb99
STABLE_BUILD_MAJOR_VERSION 1
STABLE_BUILD_MINOR_VERSION 29+
STABLE_DOCKER_TAG v1.29.0-alpha.2.1554_1850794626bb99
STABLE_DOCKER_REGISTRY registry.k8s.io
STABLE_DOCKER_PUSH_REGISTRY staging-k8s.gcr.io
gitCommit 1850794626bb995bb754d54be3328c86ee880ba5
gitTreeState clean
gitVersion v1.29.0-alpha.2.1554+1850794626bb99
gitMajor 1
gitMinor 29+
buildDate 2024-01-02T20:12:28Z
INFO Found workspace version: v1.29.0-alpha.2.1554+1850794626bb99
INFO Release path is gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99
INFO Checking if GCS build path (gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99) exists
INFO Checking if GCS build path (gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes.tar.gz) exists
INFO Checking if GCS build path (gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99/bin) exists
INFO Validating image manifests in
INFO The following error(s) occurred while looking for a build: [<nil> <nil> <nil> get remote manifest from /conformance:v1.29.0-alpha.2.1554_1850794626bb99: GET https://index.docker.io/v2/conformance/manifests/v1.29.0-alpha.2.1554_1850794626bb99: UNAUTHORIZED: authentication required; [map[Action:pull Class: Name:conformance Type:repository]]]
INFO Proceeding with a new build...
+++ [0102 20:12:45] Verifying Prerequisites....
+++ [0102 20:12:48] Removing _output directory
+++ [0102 20:12:49] Verifying Prerequisites....
+++ [0102 20:12:49] Building Docker image kube-build:build-da4d3fe969-5-v1.29.0-go1.21.5-bullseye.0
+++ [0102 20:12:51] Creating data container kube-build-data-da4d3fe969-5-v1.29.0-go1.21.5-bullseye.0
+++ [0102 20:12:51] Syncing sources to container
+++ [0102 20:12:58] Running build command...
+++ [0102 20:12:59] Building go targets for linux/amd64
    k8s.io/kubernetes/cmd/kube-proxy (static)
    k8s.io/kubernetes/cmd/kube-apiserver (static)
    k8s.io/kubernetes/cmd/kube-controller-manager (static)
    k8s.io/kubernetes/cmd/kubelet (non-static)
    k8s.io/kubernetes/cmd/kubeadm (static)
    k8s.io/kubernetes/cmd/kube-scheduler (static)
    k8s.io/component-base/logs/kube-log-runner (static)
    k8s.io/kube-aggregator (static)
    k8s.io/apiextensions-apiserver (static)
    k8s.io/kubernetes/cluster/gce/gci/mounter (static)
+++ [0102 20:19:05] Building go targets for linux/amd64
    k8s.io/kubernetes/cmd/kube-proxy (static)
    k8s.io/kubernetes/cmd/kubeadm (static)
    k8s.io/kubernetes/cmd/kubelet (non-static)
    k8s.io/component-base/logs/kube-log-runner (static)
+++ [0102 20:19:15] Building go targets for linux/amd64
    k8s.io/kubernetes/cmd/kubectl (static)
    k8s.io/kubernetes/cmd/kubectl-convert (static)
+++ [0102 20:19:30] Building go targets for linux/amd64
    github.com/onsi/ginkgo/v2/ginkgo (non-static)
    k8s.io/kubernetes/test/e2e/e2e.test (test)
    k8s.io/kubernetes/test/conformance/image/go-runner (non-static)
+++ [0102 20:23:53] Building go targets for linux/amd64
    k8s.io/kubernetes/cmd/kubemark (static)
    github.com/onsi/ginkgo/v2/ginkgo (non-static)
    k8s.io/kubernetes/test/e2e_node/e2e_node.test (test)
+++ [0102 20:25:08] Syncing out of container
+++ [0102 20:25:11] Building tarball: manifests
+++ [0102 20:25:11] Building tarball: src
+++ [0102 20:25:11] Starting tarball: client linux-amd64
+++ [0102 20:25:11] Waiting on tarballs
tar: Removing leading `/' from member names
tar: Removing leading `/' from hard link targets
tar: Removing leading `/' from member names
tar: Removing leading `/' from member names
tar: Removing leading `/' from hard link targets
tar: Removing leading `/' from member names
+++ [0102 20:25:21] Building tarball: node linux-amd64
+++ [0102 20:25:21] Building images: linux-amd64
+++ [0102 20:25:21] Starting docker build for image: kube-apiserver-amd64
+++ [0102 20:25:21] Starting docker build for image: kube-controller-manager-amd64
+++ [0102 20:25:21] Starting docker build for image: kube-scheduler-amd64
+++ [0102 20:25:21] Starting docker build for image: kube-proxy-amd64
+++ [0102 20:25:21] Starting docker build for image: kubectl-amd64
+++ [0102 20:25:26] Deleting docker image registry.k8s.io/kubectl-amd64:v1.29.0-alpha.2.1554_1850794626bb99
+++ [0102 20:25:26] Deleting docker image registry.k8s.io/kube-scheduler-amd64:v1.29.0-alpha.2.1554_1850794626bb99
+++ [0102 20:25:26] Deleting docker image registry.k8s.io/kube-proxy-amd64:v1.29.0-alpha.2.1554_1850794626bb99
+++ [0102 20:25:27] Deleting docker image registry.k8s.io/kube-controller-manager-amd64:v1.29.0-alpha.2.1554_1850794626bb99
+++ [0102 20:25:29] Deleting docker image registry.k8s.io/kube-apiserver-amd64:v1.29.0-alpha.2.1554_1850794626bb99
+++ [0102 20:25:29] Docker builds done
+++ [0102 20:25:29] Building tarball: server linux-amd64
+++ [0102 20:26:33] Building tarball: final
+++ [0102 20:26:33] Starting tarball: test linux-amd64
+++ [0102 20:26:33] Waiting on test tarballs
+++ [0102 20:26:56] Building tarball: test portable
INFO Using build version: v1.29.0-alpha.2.1554+1850794626bb99
INFO Setting BuildDir to /home/ubuntu/go/src/k8s.io/kubernetes/_output
INFO Latest version is v1.29.0-alpha.2.1554+1850794626bb99
INFO Checking bucket kubernetes-staging-a1db298f53-us-central1 for write permissions
INFO Staging local artifacts
INFO Cleaning staging dir /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99
INFO Removing /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99
INFO Creating /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99
INFO Copying release tarballs
INFO Trying to copy dir /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-tars to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-tars/kubernetes-client-linux-amd64.tar.gz to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes-client-linux-amd64.tar.gz (required: false)
INFO Copied kubernetes-client-linux-amd64.tar.gz
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-tars/kubernetes-manifests.tar.gz to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes-manifests.tar.gz (required: false)
INFO Copied kubernetes-manifests.tar.gz
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-tars/kubernetes-node-linux-amd64.tar.gz to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes-node-linux-amd64.tar.gz (required: false)
INFO Copied kubernetes-node-linux-amd64.tar.gz
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-tars/kubernetes-server-linux-amd64.tar.gz to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes-server-linux-amd64.tar.gz (required: false)
INFO Copied kubernetes-server-linux-amd64.tar.gz
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-tars/kubernetes-src.tar.gz to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes-src.tar.gz (required: false)
INFO Copied kubernetes-src.tar.gz
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-tars/kubernetes-test-linux-amd64.tar.gz to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes-test-linux-amd64.tar.gz (required: false)
INFO Copied kubernetes-test-linux-amd64.tar.gz
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-tars/kubernetes-test-portable.tar.gz to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes-test-portable.tar.gz (required: false)
INFO Copied kubernetes-test-portable.tar.gz
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-tars/kubernetes.tar.gz to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes.tar.gz (required: false)
INFO Copied kubernetes.tar.gz
INFO Copying plain binaries
INFO Copying binaries for linux platform on amd64 arch
INFO Server source found in /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64, copying them
INFO Copying server binaries from /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64
INFO Trying to copy dir /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/apiextensions-apiserver to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/apiextensions-apiserver (required: false)
INFO Copied apiextensions-apiserver
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-aggregator to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-aggregator (required: false)
INFO Copied kube-aggregator
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-apiserver to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-apiserver (required: false)
INFO Copied kube-apiserver
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-apiserver.docker_tag to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-apiserver.docker_tag (required: false)
INFO Copied kube-apiserver.docker_tag
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-apiserver.tar to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-apiserver.tar (required: false)
INFO Copied kube-apiserver.tar
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-controller-manager to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-controller-manager (required: false)
INFO Copied kube-controller-manager
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-controller-manager.docker_tag to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-controller-manager.docker_tag (required: false)
INFO Copied kube-controller-manager.docker_tag
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-controller-manager.tar to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-controller-manager.tar (required: false)
INFO Copied kube-controller-manager.tar
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-log-runner to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-log-runner (required: false)
INFO Copied kube-log-runner
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-proxy to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-proxy (required: false)
INFO Copied kube-proxy
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-proxy.docker_tag to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-proxy.docker_tag (required: false)
INFO Copied kube-proxy.docker_tag
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-proxy.tar to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-proxy.tar (required: false)
INFO Copied kube-proxy.tar
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-scheduler to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-scheduler (required: false)
INFO Copied kube-scheduler
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-scheduler.docker_tag to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-scheduler.docker_tag (required: false)
INFO Copied kube-scheduler.docker_tag
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kube-scheduler.tar to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kube-scheduler.tar (required: false)
INFO Copied kube-scheduler.tar
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kubeadm to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kubeadm (required: false)
INFO Copied kubeadm
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kubectl to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kubectl (required: false)
INFO Copied kubectl
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kubectl-convert to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kubectl-convert (required: false)
INFO Copied kubectl-convert
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kubectl.docker_tag to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kubectl.docker_tag (required: false)
INFO Copied kubectl.docker_tag
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kubectl.tar to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kubectl.tar (required: false)
INFO Copied kubectl.tar
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kubelet to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/kubelet (required: false)
INFO Copied kubelet
INFO Trying to copy file /home/ubuntu/go/src/k8s.io/kubernetes/_output/release-stage/server/linux-amd64/kubernetes/server/bin/mounter to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/bin/linux/amd64/mounter (required: false)
INFO Copied mounter
INFO Trying to copy file /tmp/source-bom-v1.29.0-alpha.2.1554+1850794626bb99.spdx to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes-source.spdx (required: false)
INFO File source-bom-v1.29.0-alpha.2.1554+1850794626bb99.spdx does not exist but is also not required
INFO Trying to copy file /tmp/release-bom-v1.29.0-alpha.2.1554+1850794626bb99.spdx to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes-release.spdx (required: false)
INFO File release-bom-v1.29.0-alpha.2.1554+1850794626bb99.spdx does not exist but is also not required
INFO Writing checksums
INFO Writing artifact hashes to SHA256SUMS/SHA512SUMS files
INFO Trying to copy file SHA256SUMS to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/SHA256SUMS (required: true)
INFO Copied SHA256SUMS
INFO Trying to copy file SHA512SUMS to /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99/SHA512SUMS (required: true)
INFO Copied SHA512SUMS
INFO Hashing files in /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99
INFO Registry is not set, will not publish container images
INFO Release path is gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99
INFO Pushing release artifacts from /home/ubuntu/go/src/k8s.io/kubernetes/_output/gcs-stage/v1.29.0-alpha.2.1554+1850794626bb99 to gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99
INFO Publishing version
INFO Version marker path is gs://kubernetes-staging-a1db298f53-us-central1/ci/fast
INFO Release path is gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99
INFO Publish version markers: [latest-fast]
INFO Publish official pointer text files to gs://kubernetes-staging-a1db298f53-us-central1/ci/fast
INFO Testing v1.29.0-alpha.2.1554+1850794626bb99 > latest-fast.txt (published)
INFO gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/latest-fast.txt does not exist but will be created
INFO Making uploaded version file public:
INFO Validating uploaded version file at https://storage.googleapis.com/kubernetes-staging-a1db298f53-us-central1/ci/fast/latest-fast.txt
INFO Version equals response
ubuntu@dev:~/go/src/k8s.io/kubernetes$ krel version
    __ __    ____     ______    __
   / //_/   / __ \   / ____/   / /
  / ,<     / /_/ /  / __/     / /
 / /| |   / _, _/  / /___    / /___
/_/ |_|  /_/ |_|  /_____/   /_____/
krel: The Kubernetes Release toolbox

GitVersion:    devel
GitCommit:     ca375e676a97720933feb27f4b3b4853d08d176c
GitTreeState:  clean
BuildDate:     2024-01-02T19:30:18
GoVersion:     go1.21.5
Compiler:      gc
Platform:      linux/amd64

```

**Comment by upodroid** [comment]
duplicate builds are skipped
```
ubuntu@dev:~/go/src/k8s.io/kubernetes$ krel ci-build --bucket kubernetes-staging-a1db298f53-us-central1 --allow-dup=false --fast
INFO Build type has been set to ci
INFO Bucket has been set to kubernetes-staging-a1db298f53-us-central1
INFO GCS root has been set to ci
INFO Current working directory: /home/ubuntu/go/src/k8s.io/kubernetes
INFO Getting workspace status
STABLE_BUILD_GIT_COMMIT 1850794626bb995bb754d54be3328c86ee880ba5
STABLE_BUILD_SCM_STATUS clean
STABLE_BUILD_SCM_REVISION v1.29.0-alpha.2.1554+1850794626bb99
STABLE_BUILD_MAJOR_VERSION 1
STABLE_BUILD_MINOR_VERSION 29+
STABLE_DOCKER_TAG v1.29.0-alpha.2.1554_1850794626bb99
STABLE_DOCKER_REGISTRY registry.k8s.io
STABLE_DOCKER_PUSH_REGISTRY staging-k8s.gcr.io
gitCommit 1850794626bb995bb754d54be3328c86ee880ba5
gitTreeState clean
gitVersion v1.29.0-alpha.2.1554+1850794626bb99
gitMajor 1
gitMinor 29+
buildDate 2024-01-02T22:27:54Z
INFO Found workspace version: v1.29.0-alpha.2.1554+1850794626bb99
INFO Release path is gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99
INFO Checking if GCS build path (gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99) exists
INFO Found gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99
INFO Checking if GCS build path (gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes.tar.gz) exists
INFO Found gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99/kubernetes.tar.gz
INFO Checking if GCS build path (gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99/bin) exists
INFO Found gs://kubernetes-staging-a1db298f53-us-central1/ci/fast/v1.29.0-alpha.2.1554+1850794626bb99/bin
INFO no image registry was supplied, assuming no images are being pushed
INFO Build already exists. Exiting...
ubuntu@dev:~/go/src/k8s.io/kubernetes$ echo $?
0
```

**Review by xmudrii — APPROVED** [review]
Just a non-blocking nit
/lgtm
/approve
/hold
if other folks want to take a look

File: pkg/build/ci.go [hunk 1]
```diff
@@ -139,21 +139,23 @@ func (bi *Instance) checkBuildExists() (bool, error) {
...
 
 	images := release.NewImages()
 	imagesExist, imagesExistErr := images.Exists(bi.opts.Registry, version, bi.opts.Fast)
 	if imagesExistErr != nil {
 		existErrors = append(existErrors, imagesExistErr)
 	}
-
-	if imagesExist && len(existErrors) == 0 {
+	// we are expecting atleast 3 items to be found; /version folder, kubernetes.tgz and /version/bin folder
+	if imagesExist && len(existErrors) == 0 && foundItems >= 3 && !bi.opts.AllowDup {
```
[the remark this answers is not in the document]
Comment (cpanato) [reply pkg/build/ci.go:158]: agree

[the remark this answers is not in the document]
Comment (cpanato) [reply pkg/build/ci.go:158]: @upodroid can you address this comment and we merge this?

thanks for doing this work

[the remark this answers is not in the document]
Comment (upodroid) [reply pkg/build/ci.go:159]: Done

**Review by cpanato — APPROVED** [review]
thanks

/lgtm

**Pull request merged**